scripts/conversion_script.py

```diff
@@ -83,6 +83,32 @@ def to_dict(self):
         }
 
 
+def check_convert2rhel_inhibitors_before_run():
+    """
+    Conditions that must be True in order to run convert2rhel command.
+    """
+    ini_report = (
+        "Custom or modified ini file would cause re-registration of system to Red Hat Insights "
+        "therefore conversion was inhibited. "
+        "Please restore the /etc/convert2rhel.ini to its original state by running "
+        "'yum reinstall convert2rhel' or uninstall convert2rhel by running "
+        "'yum remove convert2rhel' "
+        "(running conversion task again will install convert2rhel again)."
+    )
+    custom_ini_exists = os.path.exists("~/.convert2rhel.ini")
+    if custom_ini_exists:
+        raise ProcessError(
+            message="Custom ~/.convert2rhel.ini was found.", report=ini_report
+        )
+
+    _, ini_file_not_modified = run_subprocess(["/usr/bin/rpm", "-Va", "convert2rhel"])
+    if ini_file_not_modified:
```

Comment:
This would capture any change to any file owned by the convert2rhel rpm, not just the /etc/convert2rhel.ini file. And by any change I mean even a timestamp change. What we care about though is the content change (MD5 hash mismatch).
@hosekadam has been recently adding a check if content of a certain rpm file has been edited: https://github.com/oamg/convert2rhel/pull/875/files. I'm not saying we need to do it this thorough here but at least looking for only a line with `/etc/convert2rhel.ini` would be good.

Reply:
It looks good, thanks.

Reply:
Btw, @andywaltlova, I've stumbled upon you response by accident as I don't watch notifications from GitHub anymore. Except when I'm tagged - then the notification goes directly to my mail inbox. Please tag me next time :) Thanks!